src/DbSQLiteAdapter/RecordSet.cpp

```diff
@@ -9,6 +9,41 @@
 
 
 namespace systelab { namespace db { namespace sqlite {
+	
+	namespace
+	{
+		int declaredTypeToColumnTypeConversion(const std::string& type)
+		{
+			if (type.find("INT") != std::string::npos ||
```

Comment:
Is there any particular reason why we are doing a find and not a direct comparison?
Do we have the type surrounded by other information?

Reply:
Not a real reason, I think is just for covering more than one similar name at once, such as INT/INTEGER/BIGINT, DATE/DATETIME, CHAR/VARCHAR,... DML allows several types to define the fields of a table, but internally, SQLite3 only declare four of them (SQLITE_INTEGER, SQLITE_STRING, SQLITE_BLOB, SQLITE_FLOAT) and the special type SQLITE_NULL